controls/cis_rhel8.yml

```diff
@@ -0,0 +1,2268 @@
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+    rules:
+      - mount_option_nosuid_removable_partitions
+
+  - id: 1.1.20
+    title: Ensure noexec option set on removable media partitions (Manual)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+    rules:
+      - mount_option_noexec_removable_partitions
+
+  - id: 1.1.22
+    title: Disable Automounting (Automated)
+    levels:
+      - l1_server
+      - l2_workstation
+    automated: yes
+    rules:
+      - service_autofs_disabled
+
+  - id: 1.1.23
+    title: Disable USB Storage (Automated)
+    levels:
+      - l1_server
+      - l2_workstation
+    automated: yes
+    rules:
+      - kernel_module_usb-storage_disabled
+
+  - id: 1.2.1
+    title: Ensure Red Hat Subscription Manager connection is configured (Manual)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+
+  - id: 1.2.2
+    title: Disable the rhnsd Daemon (Manual)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+    related_rules:
+      - service_rhnsd_disabled
+
+  - id: 1.2.3
+    title: Ensure GPG keys are configured (Manual)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+    related_rules:
+      - ensure_redhat_gpgkey_installed
+
+  - id: 1.2.4
+    title: Ensure gpgcheck is globally activated (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - ensure_gpgcheck_globally_activated
+
+  - id: 1.2.5
+    title: Ensure package manager repositories are configured (Manual)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+
+  - id: 1.3.1
+    title: Ensure sudo is installed (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - package_sudo_installed
+
+  - id: 1.3.2
+    title: Ensure sudo commands use pty (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - sudo_add_use_pty
+
+  - id: 1.3.3
+    title: Ensure sudo log file exists (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - sudo_custom_logfile
+
+  - id: 1.4.1
+    title: Ensure AIDE is installed (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - package_aide_installed
+
+  - id: 1.4.2
+    title: Ensure filesystem integrity is regularly checked (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - aide_periodic_cron_checking
+
+  - id: 1.5.1
+    title: Ensure permissions on bootloader config are configured (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - file_owner_grub2_cfg
+      - file_groupowner_grub2_cfg
+      - file_permissions_grub2_cfg
+
```

Comment:
The benchmark does take an interesting approach here, setting the `umask` and `fmask` of the `/boot/efi` mountpoint in `/etc/fstab` rather than merely changing the permissions of the files, but I think the end-result is effectively the same and our actions here will satisfy the audit and rationale from the benchmark even if it doesn't use the exact pattern laid out in the remediation section for this control.

With that in mind, I've added the rules you suggest - which also brings RHEL 8's version of this rule into line with how you've done it for RHEL 7.